smartsim/_core/launcher/dragon/dragonBackend.py

```diff
@@ -620,7 +606,7 @@ def _refresh_statuses(self) -> None:
                         except KeyError:
                             logger.error(f"Tried to free a non-allocated host: {host}")
                         self._free_hosts.append(host)
-                    group_info.process_group = None
+                    # group_info.process_group = None
```

Comment:
don't forget to remove your commented code

Reply:
Removed!